tflite/tflite-rtsp/README.md

```diff
@@ -38,8 +59,28 @@ docker run -it --rm -p 8554:8554 \
   --name tflite-rtsp <image-tag>
 ```
 
-## To view the application output, execute
+Note: Toradex provides pre-built images for this sample, just replace **image-tag** in the run command with torizonextras/arm64v8-sample-tflite-rtsp:${CT_TAG_DEBIAN}.
+
+## Run the RTSP client
+
+On your development machine, make sure to have GStreamer installed. If you need to install, refer to [Installing GStreamer](https://gstreamer.freedesktop.org/documentation/installing/index.html?gi-language=c).
+
+Then, you are able to run one of the command below on your PC, according to the desired environmental:
```

Comment:
Just a small correction: "according to the desired environment:"

Reply:
Thanks, this was fixed!